airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -50,13 +55,13 @@ def _get_init_containers(self, volume_mounts):
             'value': self.kube_config.git_branch
         }, {
             'name': 'GIT_SYNC_ROOT',
-            'value': os.path.join(
-                self.worker_airflow_dags,
-                self.kube_config.git_subpath
-            )
+            'value': self.kube_config.git_sync_root
         }, {
             'name': 'GIT_SYNC_DEST',
-            'value': 'dags'
+            'value': self.kube_config.git_sync_dest
+        }, {
+            'name': 'GIT_SYNC_DEPTH',
```

Comment:
does this fix the issue of the revision being in the path directory?

Reply:
@odracci ^^^

Reply:
I'm sorry, I haven't seen that comment.
`GIT_SYNC_DEPTH` just clone the repo with the last commit instead of downloading the whole repo history, it speeds up the deployment.
The revision being in the path directory is how git-sync works and is not possible to remove it.
I solved the problem with `kube_config.git_sync_root` which includes the dags_folder + revision + git_sub_path
/cc @jzucker2 @dimberman